src/iconsManifest/supportedExtensions.ts

```diff
@@ -4345,5 +4345,11 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
+    {
+      icon: 'glitter',
+      extensions: ["glitterrc"]
```

Comment:
I guess that's a file name? If that's so, you need to add `filename: true` for this to work.